BookingsApi/BookingsApi.DAL/Migrations/20221026065335_AddColumnVhoNonAvailabilityTableDeletedColumns.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+#nullable disable
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddColumnVhoNonAvailabilityTableDeletedColumns : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Boolean>(
+                name: "Deleted",
+                table: "VhoNonAvailability",
+                nullable: true);
```

Comment:
rather than making this nullable, could we make it default to false instead?

Reply:
done